cpp/lib/ClientQueryContextCache.hpp

```diff
@@ -0,0 +1,60 @@
+/*
+* File:   ClientQueryContextCache.hpp
```

Comment:
could we remove other tags than Copyright?

Reply:
fixed